app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -22,6 +22,31 @@ def create
         end
       end
 
+      def create_with_override
+        plan_params = Plans::PrepareForOverrideService
+          .new(current_organization, overridden_plan_code_params[:overridden_plan_code])
+          .call(plan_params: create_with_override_plan_params)
+
+        override_service = Subscriptions::OverrideService.new
+
+        result = override_service.call_from_api(
+          organization: current_organization,
+          plan_args: plan_params,
+          subscription_args: create_with_override_subscription_params
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)